src/pages/variants/index.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { FunctionComponent } from 'react';
+import StackLayout from '@ferlab/ui/core/layout/StackLayout';
+import { Typography } from 'antd';
+import style from './Variants.module.scss';
+import SearchView from './SearchView';
+
+const { Title } = Typography;
+
+const VariantPage: FunctionComponent = () => {
+  return (
+    <StackLayout vertical className={style.mainContainer}>
+      <div className={style.titleContainer}>
```

Comment:
These title are use on all page, they shouldn't have to require so much css on each page. I think the PageContent will also fix that